### PR TITLE
docs: update openapi documentation based on current code (SDKCF-6592)

### PR DESCRIPTION
**Pull request by ts-robih-robihamanto**
# Description
Update `openapi.yml` file in the repo with the recent changes in the backend API.

## Links
SDKCF-6592

# Checklist
- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [x] I have added to the [changelog](../blob/master/CHANGELOG.md#Unreleased)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data **before every commit**, including API endpoints and keys, and internal links
- [ ] I ran `fastlane ci` without errors
- [ ] All project file changes are replicated in `SampleSPM/SampleSPM.xcodeproj` project

### CODE REVIEW

**Review by ts-dzmitry-kurski — APPROVED** [review]

File: openapi.yaml [hunk 1]
```diff
@@ -83,6 +83,11 @@ paths:
         schema:
           type: string
           example: App subscription key
+      - name: easy_id
```
Comment (imaginaris) [inline openapi.yaml]: Please remove all `easy_id` references as the SDK doesn't send them

File: openapi.yaml [hunk 2]
```diff
@@ -372,7 +409,7 @@ components:
         action:
           type: integer
           example: 3
-          description: Actions -> 0 - INVALID, 1 - Redirect, 2 - Deeplink, 3 - Close
+          description: Actions -> 0 - INVALID, 1 - Redirect, 2 - Deeplink, 3 - Close, 4 - Push Premier
```
Comment (imaginaris) [inline openapi.yaml]: typo: "Premier" -> "Primer"

File: openapi.yaml [hunk 3]
```diff
@@ -242,6 +269,14 @@ components:
                 properties:
                   campaignId: 
                     type: string
+                  maxImpressions:
```
Comment (imaginaris) [inline openapi.yaml:257]: I can't believe this basic property was missing here 😅

File: openapi.yaml [hunk 4]
```diff
@@ -333,6 +368,8 @@ components:
                             type: integer
                             description: Min time between consecutive messages. Milliseconds.
                             example: 3000
+                          html:
+                            type: boolean
```
Comment (imaginaris) [inline openapi.yaml:357]: please add `description: If true, the messageBody value is treated as HTML content`

**Review by imaginaris — CHANGES_REQUESTED** [review]

File: openapi.yaml [hunk 5]
```diff
@@ -124,6 +124,17 @@ paths:
         schema:
           type: string
           example: App subscription key
+      - name: easy_id
```
Comment (imaginaris) [inline openapi.yaml]: please remove this entry as well

File: openapi.yaml [hunk 6]
```diff
@@ -169,6 +180,11 @@ paths:
           type: string
           format: uuid
           example: Device id of user's device
+      - name: easy_id
```
Comment (imaginaris) [inline openapi.yaml]: please remove this entry as well

**Review by imaginaris — APPROVED** [review]

**Pull request merged**